lambda/src/main/scala/pricemigrationengine/handlers/NotificationHandler.scala

```diff
@@ -172,7 +172,14 @@ object NotificationHandler extends CohortHandler {
       case ShouldProceed => ZIO.succeed(())
       case ShouldCancel => {
         val result = CancelledAmendmentResult(item.subscriptionName)
-        ZIO.succeed(CohortTable.update(CohortItem.fromCancelledAmendmentResult(result)).as(result))
+        ZIO.succeed(
+          CohortTable
+            .update(
+              CohortItem
+                .fromCancelledAmendmentResult(result, "(cause: f5c291b0) Subscription was cancelled by RateplansProbe")
```

Comment:
```suggestion
                .fromCancelledAmendmentResult(result, "(cause: f5c291b0) Migration cancelled by RateplansProbe")
```

less ambiguous wording - "subscription" and "cancelled" in a sentence usually has a very specific meaning

Reply:
Love it, thanks! And you are right the word "Subscription" was not the right one here, should have been "Cohort Item" at least :)